.github/scripts/build.sh

```diff
@@ -2,10 +2,15 @@
 
 source `dirname ${BASH_SOURCE[0]}`/config-mingw.sh
 
+.github/scripts/install-dependencies.sh
```

Comment:
I use this script often locally. it will be preferable to have dependencies and library installation disabled by default.

Reply:
Makes sense, I'll add `RUN_BOOTSTRAP` variable to config.sh defaulting to false.